content/slides/open-science-and-data.md

```diff
@@ -53,6 +53,20 @@ title-slide-attributes:
 <!-- alex enable easy -->
 
 
+## FAIR: DataOne
+
+Datasets registered with DataOne include a _Metadata Assessment Report_ that
+rates the dataset on FAIR complianace.
+
+![DataOne FAIR assessment](/_media/data_one_fair_assessment.png)
+
+::: {.notes}
+The Arctic Data Center, as a participating DataOne repository, also has FAIR
+data assessments. Additionally, the ADC has it's own assessment suite.
```

Comment:
Does the ADC's own assessment suite have a name or URL? Would participants be able to leverage it locally?

Reply:
It's just called the Arctic Data Center assessment suite. NOt sure about a URL or leveraging locally.